app/views/shared/vacancy/_jobseeker_view.html.haml

```diff
@@ -24,14 +24,14 @@
             = t('jobs.job_details')
         %li.govuk-tabs__list-item
           %a.govuk-tabs__tab.tab-links-override{ href: "#school-overview" }
-            - if @vacancy.school_group
+            - if @vacancy.job_location == 'central_office'
               = t('school_groups.trust_overview')
             - else
               = t('schools.school_overview')
 
       = render(Jobseekers::VacancyDetailsComponent.new(vacancy: @vacancy))
 
-      - if @vacancy.school_group
+      - if @vacancy.job_location == 'central_office'
```

Comment:
Since you're here..... and you've opened the can of worms...

May I request we remove the conditional logic from the template, and utilise the `render?` method in the respective ViewComponents (and add relevant tests to each component) please?